vmm/src/cpu.rs

```diff
@@ -207,6 +208,12 @@ pub enum Error {
     #[cfg(target_arch = "x86_64")]
     #[error("Failed to inject NMI")]
     NmiError(#[source] hypervisor::HypervisorCpuError),
+
+    /// Cannot clean init vcpu TLS.
+    #[error("Failed to initialise vCPU TLS")]
+    VcpuTlsInit,
+    #[error("vCPU TLS not present")]
```

Comment:
I think we should add a little more context, something like

```suggestion
    #[error("vCPU was moved to another thread early and TLS is no longer present")]
```